xwiki-commons-core/xwiki-commons-xml/src/main/java/org/xwiki/xml/XMLUtils.java

```diff
@@ -276,15 +277,15 @@ public static String escapeAttributeValue(Object content)
     }
 
     /**
-     *  Escapes some XML special characters (&amp; and &lt;) in a <code>String</code> using numerical XML
-     *  entities, so that the resulting string can safely be used in HTML pages.
+     * Escapes XML special characters in a {@code String} using numerical XML entities, so that the resulting string
+     * can safely be used as an XML text value. Specifically, escapes &lt;, and &amp;.
      *
      * @param content the text to escape, may be {@code null}
      * @return a new escaped {@code String}, {@code null} if {@code null} input
      * @since 12.8RC1
      */
     @Unstable
-    public static String minimalEscape(Object content)
+    public static String escapeElementText(Object content)
```

Comment:
I think the javadoc for the content param should also be improved and we should explain why we accept an Object and not a String and what we do to it (say that we convert it to a String).

Reply:
Documentation of the content parameter updated here https://github.com/xwiki/xwiki-commons/pull/102/files#diff-c1eed56adbad8fd67a8e196907ce032fR292